packages/run-it/src/components/PerfTracker/PerfChart.tsx

```diff
@@ -84,33 +86,36 @@ export const PerfChart: FC<PerfChartProps> = ({ loadTimes }) => {
       <Heading as="h4">{loadTimes.name}</Heading>
       <Space>
         <Tooltip
-          content="The size (in octets) of the fetched resource. The size includes the response header fields plus the response payload body"
+          content={`The size (${entry.transferSize} octets) of the fetched resource. The size includes the response header fields plus the response payload body`}
           placement="right"
           textAlign="left"
         >
-          <span>Transfer size: {entry.transferSize}</span>
+          <span>Transfer: {entry.transferSize}</span>
         </Tooltip>
         <Tooltip
-          content="The size (in octets) received from the fetch (HTTP or cache), of the payload body, before removing any applied content-codings"
+          content={`The size (${entry.encodedBodySize} octets) received from the fetch (HTTP or cache), of the payload body, before removing any applied content-codings`}
           placement="right"
           textAlign="left"
         >
-          <span>Encoded body size: {entry.encodedBodySize}</span>
+          <span>Encoded body: {entry.encodedBodySize}</span>
         </Tooltip>
         <Tooltip
-          content="The size (in octets) received from the fetch (HTTP or cache) of the message body, after removing any applied content-codings"
+          content={`The size (${entry.decodedBodySize} octets) received from the fetch (HTTP or cache) of the message body, after removing any applied content-codings`}
           placement="right"
           textAlign="left"
         >
-          <span>Decoded body size: {entry.decodedBodySize}</span>
+          <span>Decoded body: {entry.decodedBodySize}</span>
         </Tooltip>
       </Space>
       <Chart
         width={'100%'}
-        height={'350px'}
+        height={`${8 * 41 + 10}px`}
```

Comment:
Still scrolling. 418 should do it. Also, a number is accepted.

Reply:
Sorry, I don't understand what _418 should do it_ means